src/main/webapp/app/course/competencies/competencies-popover/competencies-popover.component.html

```diff
@@ -1,11 +1,11 @@
 <ng-template #popTitle> <span class="font-weight-bold" jhiTranslate="artemisApp.competency.competencyPopover.connectedCompetencies"></span></ng-template>
 <ng-template #popContent>
-    @if (competencies && competencies.length > 0) {
+    @if (competencyLinks?.length) {
         <div>
             <small jhiTranslate="artemisApp.competency.competencyPopover.explanation"></small>
             <ul class="list-group">
-                @for (competency of competencies; track competency) {
-                    <a class="list-group-item" [routerLink]="navigationArray">{{ competency.title }}</a>
+                @for (competencyLink of competencyLinks; track competencyLink) {
+                    <a class="list-group-item" [routerLink]="navigationArray">{{ competencyLink.competency?.title }}</a>
```

Comment:
Would it make sense to wrap the link in an if instead of the optional chaining?
If the title is not there the link will most likely look quite broken